src/chrome/chromeUtils.ts

```diff
@@ -16,6 +16,7 @@ import { IResourceIdentifier } from './internal/sources/resourceIdentifier';
 import { isNotEmpty, isEmpty, isDefined, hasMatches, isUndefined, defaultWhenEmpty } from './utils/typedOperators';
 import * as _ from 'lodash';
 import { notEmpty } from '../validation';
+import isNull = require('lodash/isNull');
```

Comment:
Do we have to use `require`? It has types right?

Reply:
We can use import, and that was the wrong library.
I wrote some custom code because our existing functions don't work for this particular case